server/bundler/loader.js

```diff
@@ -44,7 +44,17 @@ function getSectionsModule( sections ) {
 			'	load: function() {',
 			'		' + sectionLoaders,
 			'	}',
-			'};'
+			'};',
+			'function loadCSS( section ) {',
+			'	if ( section.css ) {',
+			'		var link = document.createElement( "link" );',
+			'		link.setAttribute( "rel", "stylesheet" );',
+			'		link.setAttribute( "type", "text/css" );',
+			'		link.setAttribute( "href", "/calypso/sections/" + section.css + ".css" );', // TODO hashing
```

Comment:
We'll need to add a hash so that we don't hit cached files. We'll also need to make sure that we load the correct RTL or debug versions of the files here.